packages/cfa-template/template/backend/app.js

```diff
@@ -41,8 +41,8 @@ const createApp = (applyMiddleware) => {
     return app;
 };
 
-async function createAppHandler(eventName, router, shouldUseDatabase = true) {
-    await secretsToEnv();
+ function createAppHandler(eventName, router, shouldUseDatabase = true) {
+     secretsToEnv().catch((err) => { console.error('Error fetching secrets:', err);  });
```

Comment:
Don't we need to finish loading the secrets before creating the app?

Reply:
Removed 👍